wgs/workflows/postprocessing/__init__.py

```diff
@@ -5,12 +5,27 @@
 import os
 
 def get_coverage_data(input_bam, output, refdir, single_node=False):
-    refdir_paths = config.refdir_data(refdir)['paths']
     chromosomes = config.refdir_data(refdir)['params']['chromosomes']
+    bins = config.refdir_data(refdir)['params']['bins']
+
+    reference = config.refdir_data(refdir)['paths']['reference']
 
     workflow = pypeliner.workflow.Workflow()
 
     if single_node:
+        workflow.transform(
+            name='generate_coverage_bed',
+            func='wgs.workflows.postprocessing.tasks.generate_coverage_bed',
+            ctx=helpers.get_default_ctx(
+                memory=5
+            ),
+            args=(
+                reference,
+                mgd.TempOutputFile('coverage_bed.bed'),
+                chromosomes,
+                bins,
+            )
+        )
         workflow.transform(
             name='samtools_coverage',
```

Comment:
ideally you would write this to use multiple cores but on the same machine.
example: https://github.com/shahcompbio/wgs/blob/master/wgs/utils/museq_utils.py#L66

Reply:
ok....if it is a speed boost that doesn't matter this is essentially ~instant when n bins is not in the  millions

Reply:
I'm sure this coverage code takes a while to run on a full chromosome?


how long would it take to run this on a full genome in a single job serially??